doc/classes/ProjectSettings.xml

```diff
@@ -1024,6 +1024,11 @@
 			Sets which physics engine to use for 3D physics.
 			"DEFAULT" is currently the [url=https://bulletphysics.org]Bullet[/url] physics engine. The "GodotPhysics" engine is still supported as an alternative.
 		</member>
+		<member name="physics/common/delta_smoothing" type="bool" setter="" getter="" default="false">
+			Time samples for frame deltas are subject to random variation introduced by the platform, even when frames are displayed at regular intervals due to vsync. This can lead to jitter. Delta smoothing can often give a better result by filtering the input deltas to correct for minor fluctuations from the refresh rate.
+			[b]Note:[/b] Delta smoothing is only attempted when [member display/window/vsync/use_vsync] is switched on, it does not work well without vsync.
+			It may take some time (several seconds) at a stable frame rate before the smoothing is initially activated. It will only be active on machines where performance is adequate to render frames at the refresh rate.
```

Comment:
```suggestion
			Time samples for frame deltas are subject to random variation introduced by the platform, even when frames are displayed at regular intervals thanks to V-Sync. This can lead to jitter. Delta smoothing can often give a better result by filtering the input deltas to correct for minor fluctuations from the refresh rate.
			[b]Note:[/b] Delta smoothing is only attempted when [member display/window/vsync/use_vsync] is switched on, as it does not work well without V-Sync.
			It may take several seconds at a stable frame rate before the smoothing is initially activated. It will only be active on machines where performance is adequate to render frames at the estimated refresh rate.
```

Reply:
Have changed these except the last one, 'estimated refresh rate'. I don't think that wording would be good - as it would be incorrect, because in the source code the estimated frame rate is a distinct thing, and the estimation may at any point in time be incorrect. What I'm referring to here is the actual refresh rate of the monitor. (The situation is a bit more complex with a variable refresh rate monitor, but the system should hopefully not activate in that case)